library/src/main/res/layout/mcab_toolbar.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.v7.widget.Toolbar
+<androidx.appcompat.widget.Toolbar
```

Comment:
This is a bit of a large change for a production library right now.

Reply:
But android.support.v7.widget.Toolbar is not present anymore in androidX support library

Reply:
@marbat87 yes but it's experimental and requires flags to work, Material CAB is not

Reply:
@afollestad I don't understand. Is it part of new design library and it is a RC version.. so it should be stable.
"This release candidate of AndroidX is considered feature-complete and its public API surface is stable"

Reply:
@afollestad Not sure if you've had time to look into androidx, but there's nothing experimental about it.  Support libs are dead and technically Material Cab is EoL unless it updates to androidx.  It will more than likely mean 2.0.0, but anyone properly managing their semver and dodging the inevitable androidx migration should be fine.

And yes, migration can be a pain in the ass.